go.mod

```diff
@@ -42,20 +43,20 @@ require (
 	github.com/rancher-sandbox/ele-testhelpers v0.0.0-20221213084338-a8ffdd2b87e3
 	github.com/rancher-sandbox/go-tpm v0.0.0-20220823075603-d273b298fcda
 	github.com/rancher/fleet/pkg/apis v0.0.0-20220318160658-4dc66c946ca2
-	github.com/rancher/lasso v0.0.0-20210709145333-6c6cd7fd6607
+	github.com/rancher/lasso v0.0.0-20220628160937-749b3397db38
 	github.com/rancher/rancher/pkg/apis v0.0.0-20211013185633-a636bda2a00e
 	github.com/rancher/steve v0.0.0-20220503004032-53511a06ff37
-	github.com/rancher/system-agent v0.2.8
+	github.com/rancher/system-agent v0.2.14-0.20230106171935-24c523a4408a
 	github.com/rancher/system-upgrade-controller/pkg/apis v0.0.0-20220321230925-79da9f058209
-	github.com/rancher/wrangler v0.8.11-0.20220217210408-3ecd23dfea3b
-	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v1.5.0
+	github.com/rancher/wrangler v1.0.0
```

Comment:
I wonder why this is still around 🤔

Reply:
Hmm, looks like it's being used in `pkg/dmidecode` and `controllers` packages.